srcpkgs/numen/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'numen'
+pkgname=numen
+version=0.5
+revision=1
+build_style=go
+go_import_path="git.sr.ht/~geb/numen"
+hostmakedepends="scdoc"
+makedepends="vosk-api libgomp-devel"
+depends="alsa-utils dmenu dotool xdotool xset"
+short_desc="Voice control for handsfree computing"
+maintainer="John Gebbie <me@johngebbie.com>"
+license="GPL-3.0-only"
+homepage="https://numenvoice.com"
+changelog="https://git.sr.ht/~geb/numen/blob/master/CHANGELOG.md"
+distfiles="https://git.sr.ht/~geb/numen/archive/${version}.tar.gz"
+checksum=6b1ca7b5b9b2134d43ce4f684b3833d75be2de66354e302d43ec2d03b071a520
+conf_files="/etc/sv/numen/conf"
+
+if [ "$XBPS_TARGET_LIBC" = musl ]; then
+	makedepends+=" libexecinfo-devel"
+fi
+
+system_accounts="_numen"
+
+pre_build() {
+	# for the vosk-api package (openblas)
+	export CGO_LDFLAGS="$CGO_LDFLAGS -fopenmp"
+
+	if [ "$XBPS_TARGET_LIBC" = musl ]; then
+		# for the vosk-api package (openfst)
+		export CGO_LDFLAGS="$CGO_LDFLAGS -lexecinfo"
+	fi
+}
+
+post_install() {
+	mv "${DESTDIR}/usr/bin/numen" speech
```

Comment:
what's the purpose of this line?

Reply:
numen is not just a go executable, it's a pipline with shell scripts too.
The go module name has to be `git.sr.ht/~geb/numen` but the go file is speech.go.
I have added `go_package="speech.go"` so it is compiled as `speech` which is a slight improvement I think.

(I also thought I could compile it to $wrksrc by with `do_install` instead of `post_install` but then it ends up in a directory in $GOPATH.)